wandb/run_manager.py

```diff
@@ -925,6 +925,11 @@ def init_run(self, env=None):
                 # Currently we assume we're not resuming in the case of resume = auto,
                 # and we throw an error in the case of resume = must.
                 logger.info("checking resume status, waiting at most %d seconds" % InternalApi.HTTP_TIMEOUT)
+
+                if not self._project:
+                    raise LaunchError(
+                        "resume='must' but no project is currently specified, run `wandb init` to set up this project")
```

Comment:
I think we should tell users to specify `project="..."` in their call to wandb.init instead of running `wandb init`.

Reply:
Good call. Done.